otx/algorithms/detection/adapters/mmdet/nncf/compression.py

```diff
@@ -236,6 +205,8 @@ def model_eval_fn(model):
         nncf_compress_postprocessing = cfg.get('nncf_compress_postprocessing')
         logger.debug('set should_compress_postprocessing='f'{nncf_compress_postprocessing}')
     else:
+        # TODO: Do we have to keep this configuration?
+        # This configuration is not enabled in forked mmdetection library in the first place
         nncf_compress_postprocessing = True
 
```

Comment:
Seems that there is no setting value in config files so that it means it is always True.
Do you mean that we can remove this config and treat it as True always?

Reply:
In forked mmdet repository, when building NNCF, it always does not postprocess.
Please refer: https://github.com/openvinotoolkit/mmdetection/blob/ote/mmdet/models/detectors/base.py#L231
In `forward_export` method, it always calls `simple_test` method without postprocessing.

Whatever `nncf_compress_postprcessing` is, it always do not compress postprocessing part in mmdet.
So [forward_dummy](https://github.com/openvinotoolkit/mmdetection/blob/ote/mmdet/models/detectors/single_stage.py#L68-L75) and [forward_export](https://github.com/openvinotoolkit/mmdetection/blob/ote/mmdet/models/detectors/single_stage.py#L107-L124) do the same thing when tracing graph.